server/lyft/decorators/destroy_plan_step_runner_wrapper.go

```diff
@@ -0,0 +1,21 @@
+package decorators
+
+import (
+	"github.com/runatlantis/atlantis/server/events"
+	"github.com/runatlantis/atlantis/server/events/models"
+)
+
+const Deprecated = "deprecated"
+const Destroy = "-destroy"
+
+type DestroyPlanStepRunnerWrapper struct {
+	events.StepRunner
+}
+
+func (d *DestroyPlanStepRunnerWrapper) Run(ctx models.ProjectCommandContext, extraArgs []string, path string, envs map[string]string) (string, error) {
+	// DestroyPlan tag is true when the Terraform client should construct a destroy plan given a repo config.
+	if ctx.Tags[Deprecated] == Destroy {
```

Comment:
Sorry, one more nit. You can just use a flag instead of explicit `-destroy` value.

Reply:
looks like tags are string values, we can keep it as is